Resources/config/twig.xml

```diff
@@ -8,5 +8,8 @@
         <service id="sonata.user.twig.global" class="Sonata\UserBundle\Twig\GlobalVariables" >
             <argument type="service" id="service_container" />
         </service>
+        <service id="sonata.user.twig.extentions" class="Sonata\UserBundle\Twig\SecurityExtentions">
```

Comment:
extenTion??? shoudl be extenSion

and

Sonata\UserBundle\Twig\SecurityExtentions should be Sonata\UserBundle\Twig\SecurityExtensions